web/components/ChatMessage.js

```diff
@@ -1,53 +1,54 @@
 /**
- * Copyright (c) 2018, salesforce.com, inc.
+ * Copyright (c) 2020, salesforce.com, inc.
  * All rights reserved.
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or
  * https://opensource.org/licenses/BSD-3-Clause
  */
 
+import React from 'react';
+import linkifyHtml from 'linkifyjs/string';
+import decode from 'unescape';
 import PropTypes from 'prop-types';
 import ReactHtmlParser from 'react-html-parser';
+import EventBlueprint from './EventBlueprint';
 import './chat.css';
 
-const React = require('react');
-const linkifyHtml = require('linkifyjs/string');
-const decode = require('unescape');
-import EventBlueprint from './EventBlueprint';
-const sldsBlue = '#34becd';
+const SLDS_BLUE = '#34becd';
```

Comment:
should we move all these to their own file ?

Reply:
That's a good point, makes sense